chapter03/qli/src/main/java/spring/training/beanConfig/BankBeanConfig.java

```diff
@@ -0,0 +1,48 @@
+package spring.training.beanConfig;
```

Comment:
Spring is used only in the testing. I feel, this configuration can be moved into the test folder.